tests/storage/test_cdi_resources.py

```
@@ -46,18 +46,21 @@ def verify_label(cdi_resources):
 
 
 def verify_cdi_app_label(cdi_resources, cnv_version):
     for resource in cdi_resources:
-        if resource.kind == "Secret" and resource.name not in CDI_SECRETS:
-            continue
-        elif resource.kind == "ServiceAccount" and resource.name == CDI_OPERATOR:
-            continue
-        elif resource.kind == "ConfigMap" and resource.name not in CDI_CONFIGMAPS:
+        if (
+            resource.kind == "Secret"
+            and resource.name not in CDI_SECRETS
```

Ruff (0.15.12)

[warning] Parenthesize `a and b` expressions when chaining `and` and `or` together, to make the precedence clear

Parenthesize the `and` subexpression

(RUF021)